flow/skia_gpu_object.h

```diff
@@ -60,14 +62,22 @@ class UnrefQueue : public fml::RefCountedThreadSafe<UnrefQueue<T>> {
   void Drain() {
     TRACE_EVENT0("flutter", "SkiaUnrefQueue::Drain");
     std::deque<SkRefCnt*> skia_objects;
+#if !SLIMPELLER
```

Comment:
The Dart VM has a similar problem with conditional compilation for product vs. non-product builds. They use macros like these, which might make this code a bit easier to read: https://github.com/dart-lang/sdk/blob/main/runtime/vm/globals.h#L80.

Reply:
I was hoping to back out SkiaGPUObject entirely but yeah, this makes sense in the meantime. It will be useful elsewhere as well. Patching.

Reply:
I added and used that pattern in a few spots.  But it was somewhat less useful that I had hoped in the eliding specific function arguments because of macro substitution rules. I saw the Dart VM running into similar limitations.

Ultimately, I want to get rid of the the ifdef using stubs most likely. But this is a start.